src/main/resources/edu/cornell/kfs/core/documentstore/InstitutionPreferences.json

```diff
@@ -602,7 +607,7 @@
               "templateNamespace": "KR-NS",
               "templateName": "Look Up Records",
               "details": {
-                "namespaceCode": "KFS*"
+                "componentName": "BudgetConstructionPosition"
```

Comment:
Based on possible changes from the other related SQL review, maybe this becomes KFS-BC?